astropy/tests/command.py

```diff
@@ -205,35 +205,43 @@ def run(self):
 
     def _build_temp_install(self):
         """
-        Build the package and copy the build to a temporary directory for
-        the purposes of testing this avoids creating pyc and __pycache__
-        directories inside the build directory
+        Install the package and to a temporary directory for the purposes of
+        testing this avoids creating pyc and __pycache__ directories inside the
+        build directory
```

Comment:
I think the pyc/pycache mention can be removed, since the goal is now to test also the `install` step.

Reply:
Actually this is still relevant but I also expanded this to mention the other benefits.